plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeFileOperations.java

```diff
@@ -378,6 +378,47 @@ public void testSelfJoin()
         assertUpdate("DROP TABLE test_self_join_table");
     }
 
+    @Test
+    public void testSelectFromVersionedTable()
+    {
+        String tableName = "test_select_from_versioned_table";
+        assertUpdate("CREATE TABLE " + tableName + " (id int, age int)");
+        long v1SnapshotId = 0;
+        assertUpdate("INSERT INTO " + tableName + " VALUES (2, 20)", 1);
+        long v2SnapshotId = 1;
```

Comment:
Can you pls add a test in a table containing multiple snapshots (register table would probably be more fit) to showcase the file accesses made on different version of of the table.

e.g.  0CreateTable    10Checkpoint       20Checkpoint 25LastCommit

Showcase the file accesses for v8, v13, v20